parser/src/parser/grammar.lalrpop

```diff
@@ -58,6 +60,22 @@ AuxCols: Vec<Identifier> = {
     "aux" ":" "[" <aux_cols: CommaElems<Identifier>> "]" => aux_cols,
 }
 
+// CONSTANTS
+// ================================================================================================
+
+// Constants section is optional but cannot be empty. Atleast one constant is required.
```

Comment:
nit: "At least" instead of "Atleast". Also, I would clarify this a bit by saying "If the section is declared, then at least one constant is required." I would also say "The constants section" instead of "Constants section"